domain/infosync/info.go

```diff
@@ -79,18 +79,21 @@ type ServerVersionInfo struct {
 	GitHash string `json:"git_hash"`
 }
 
-// NewInfoSyncer return new InfoSyncer. It is exported for testing.
-func NewInfoSyncer(id string, etcdCli *clientv3.Client) *InfoSyncer {
-	return &InfoSyncer{
+var globalInfoSyncer *InfoSyncer
+
+// GlobalInfoSyncerInit return new InfoSyncer. It is exported for testing.
+func GlobalInfoSyncerInit(ctx context.Context, id string, etcdCli *clientv3.Client) (*InfoSyncer, error) {
+	globalInfoSyncer = &InfoSyncer{
```

Comment:
`globalInfoSyncer` is a global variable, maybe something like `sync.Once` should be used?

Reply:
No, `globalInfoSyncer` can't only initialize once, It should keep consistency with `domain.ddl`.  And the `DDL` will changed after bootstrap when the tidb-cluster first initialization.